### DISCUSSION TITLE
How to get a list of receivers and CC

### DISCUSSION THREAD

**Discussion by AntonioJCosta** [O365/python-o365#803] [Q&A] [answered]
Hello. I am trying to get a list of address mail from the message, but I just found these two methods, that don't show de addresses, instead it shows the number of each one. 

cc: Recipients count: 2
to: Recipients count: 1

**Comment by alejcas** [comment, marked as answer]
Just iterate cc

```python
for address in message.cc:
   print(address)
```

**Reply by AntonioJCosta** [reply]
Thank you so much for the help!
